doc/Comments.xml

```diff
@@ -302,6 +302,25 @@ DeclareOperation( "GroupedOperation",
 ]]></Listing>
 </Subsection>
 
+<Subsection Label="@GroupTitle">
+<Index Key="@GroupTitle"><C>@GroupTitle</C></Index>
+<Heading>@GroupTitle <A>title</A></Heading>
+Sets the subsection heading for the current group to <A>title</A>. In the
+absence of any <C>@GroupTitle</C> command, the heading will be the name of the
+first entry in the group. See <Ref Sect="Groups" /> for more information.
+</Subsection>
+
+<Subsection Label="@GroupInitialArguments">
+<Index Key="@GroupInitialArgumentss"><C>@GroupInitialArguments</C></Index>
+<Heading>@GroupInitialArguments <A>args</A></Heading>
+Sets the common initial arguments for the entries in the current group to be
+<A>args</A>. The argument list for any function subsequently added to the
+group without any args specified will be <A>args</A>, and <A>args</A> will be
+prepended to any arguments specified for a particular function. The setting is
+cleared by an <C>@EndGroup</C> command. See <Ref Sect="Groups" /> for more
+information.
```

Comment:
I don't like `@GroupInitialArguments` at all. It makes information non-local: with it, when reading a function's AutoDoc comment, I must be aware of whether it is inside a group, and whether a `@GroupInitialArguments` was specified or not.

This does not seem to justify the (IMHO very minor, if at all) "savings" provided by it. It's not that difficult to repeat the string `order, ` two or three times (it won't be more in general).